test/jet/fn.clj

```diff
@@ -0,0 +1 @@
+#(-> % :a :b :c)
```

Comment:
Added this as a test function file.

Not sure if this is the best location. Please suggest an alternative if you can think of a better place.

Thanks

Reply:
I think creating a `test-resources` dir is better since these kinds of files may get executed by a test runner. In this case it doesn't hurt, but let's start with giving the right direction.

Reply:
Sure!

Reply:
Done.

Added `test-resources` at the top level of the repo.